### PR TITLE
feat: Adding Zside Port Service Token resource

### PR DESCRIPTION
**Pull request by srushti-patl**
-Added Zside Port Service Token Resource Acc Tests
- Updated docs for Service Token Resource

### CODE REVIEW

File: docs/resources/fabric_service_token.md [hunk 1]
```diff
@@ -41,6 +41,35 @@ resource "equinix_fabric_service_token" "test" {
 }
 ```
 
+Aside Port Service Token
+```terraform
+resource "equinix_fabric_service_token" "test"{
+  type = "VC_TOKEN"
+  description = "Zside COLO Service Token"
```
Comment (thogarty) [inline docs/resources/fabric_service_token.md:48]: This corresponds to Zside. The attribute z_side is given in the body of this resource too. Is this the correct implementation?

File: internal/resources/fabric/service_token/resource_test.go [hunk 2]
```diff
@@ -110,6 +110,53 @@ func TestAccFabricAsidePortServiceToken_PNFV(t *testing.T) {
 	})
 }
 
+func TestAccFabricZsidePortServiceToken_PNFV(t *testing.T) {
```
Comment (thogarty) [inline internal/resources/fabric/service_token/resource_test.go:113]: You're adding tests for Zside, but then have documentation related to Aside. I'm confused how this is working.

Comment (srushti-patl) [reply internal/resources/fabric/service_token/resource_test.go:113]: I added the wrong name in the documentation. It's zside service token example. Apologize for the confusion.

File: docs/resources/fabric_service_token.md (hunk 1 above)
Comment (srushti-patl) [reply docs/resources/fabric_service_token.md:48]: Title of this example was wrong

**Pull request merged**